src/labkey/security/Container.ts

```diff
@@ -77,6 +79,7 @@ export function deleteContainer(config: DeleteContainerOptions): XMLHttpRequest
     return request({
         url: buildURL('core', 'deleteContainer.api', config.containerPath),
         method: 'POST',
+        jsonData: {comment: config.comment},
```

Comment:
Might be worth considering only propagating the comment to the payload if a comment is actually provided.

Reply:
Noted for future implementations! This one left as is.